tests/performance/Dockerfile

```diff
@@ -1,6 +1,6 @@
 
 ARG LOCUST_VERSION
-FROM locustio/locust:${LOCUST_VERSION}
+FROM locustio/locust:${LOCUST_VERSION:-2.29.1}
```

Comment:
does this make sense to pin it there? where is LOCUST_VERSION defined?

Reply:
I answer below